CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change log
 
+## 2.1.0 (2023-08-17)
+- Updated to include .NET 6
```

Comment:
This was only test projects, we did not change the TFM of the library itself.

I'd either drop this entirely (since it doesn't matter to users), or else clarify that it was only test projects.

Reply:
Done. Thanks!